internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixReadOnlyDarkClusterConfigRepository.java

```diff
@@ -0,0 +1,102 @@
+package com.linkedin.venice.helix;
+
+import static com.linkedin.venice.zk.VeniceZkPaths.DARK_CLUSTER_CONFIG;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import com.linkedin.venice.meta.DarkClusterConfig;
+import com.linkedin.venice.meta.ReadOnlyDarkClusterConfigRepository;
+import com.linkedin.venice.utils.HelixUtils;
+import com.linkedin.venice.utils.RedundantExceptionFilter;
+import java.nio.file.Paths;
+import org.apache.helix.AccessOption;
+import org.apache.helix.manager.zk.ZkBaseDataAccessor;
+import org.apache.helix.zookeeper.impl.client.ZkClient;
+import org.apache.helix.zookeeper.zkclient.IZkDataListener;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+/**
+ * This class is used to configure dark cluster.
+ */
```

Comment:
Could you please add more information here? Like at least mention which path to look for - ${@com.linkedin.venice.zk.VeniceZkPaths.DARK_CLUSTER_CONFIG}